release/release.sh

```diff
@@ -115,3 +118,10 @@ if [ "$release_apt_and_yum" = "true" ]; then
 else
     echo "WARNING: there is no staging environment for apt or yum repos"
 fi
+
+if [ "$release_ami" = "true" ]; then
+  "$earthly" --push --build-arg RELEASE_TAG ./ami+update-pipelines
+  "$earthly" --push --build-arg RELEASE_TAG ./ami+build-ami
```

Comment:
The `--build-arg` syntax is deprecated.

```suggestion
  "$earthly" --push ./ami+update-pipelines --RELEASE_TAG="$RELEASE_TAG"
  "$earthly" --push ./ami+build-ami --RELEASE_TAG="$RELEASE_TAG"
```

Reply:
Yeah I was matching the rest of the script using the old style still.